client/src/components/Header/Header.css

```
@@ -1,6 +1,155 @@
-header > nav > ul {
-  display: flex;
-  list-style: none;
-  gap: 10px;
-  padding-left: 0; /* reset */
-}
\ No newline at end of file
+/* * {
+	border: 1px dotted black;
+} */
+
+header {
+	background-color: var(--gray80);
+}
+
+header nav {
+	position: fixed;
+	top: 0;
+	left: 0;
+	height: 100vh;
+	width: 100vw;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background-color: #adadade9;
+	transform: translateY(-100vh);
+}
+
+header nav ul {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	gap: 16px;
+	list-style: none;
+	padding-left: 0; /* reset */
+}
+
+/* disabling scrolling while mobile navbar is opened */
+body.open-nav {
+	overflow: hidden;
+}
+
+.header-subcontainer {
+	max-width: 1200px;
+	margin: 0 auto;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 0 20px;
+	height: 80px;
+}
+
+.navlink {
+	color: black;
+	text-decoration: none;
+	font-size: 14px;
+	transition: all ease-in-out 200ms;
+}
+
+.navlink:hover {
+	color: white;
+}
+
+.navlink.active {
+	color: white;
+}
+
+.navlink-image-container {
+	display: none; /* Hide the arrow by default */
+}
+
+/* hamburger button */
+.nav-btn {
+	padding: 5px;
+	cursor: pointer;
+	background: transparent;
+	border: none;
+	outline: none;
+	font-size: 1.8rem;
+	display: block;
+	color: black;
+}
+
+/* close mobile menu button */
+.nav-close-btn {
+	position: absolute;
+	top: 2rem;
+	right: 2rem;
```

This centers the X directly over the 3 Lines (Hamburger) 
(Please check on your end)

```suggestion
	top: 18px;
	right: 20px;
```